src/dbSta/src/dbNetwork.cc

```diff
@@ -2562,4 +2607,159 @@ dbNetworkObserver::~dbNetworkObserver()
   }
 }
 
+/*
+  Hierarchical network api connections
+ */
+
+dbModule* dbNetwork::getParentModule(dbNet* net)
```

Comment:
A dbNet can span multiple modules.  The parent here seems to be the parent of the driver.  Is that always what is wanted?  If so the name should reflect that.

Reply:
Yes. The parent of the driver. Ok will call getDriverParentModule.

Reply:
not resolved

Reply:
done, called getNetDriverParentModule. (gets the module parent of the net driver)